functions/helloworld/helloworld/pom.xml

```diff
@@ -16,22 +16,27 @@
   limitations under the License.
 -->
 
+<!-- [START functions_example_pom] -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.example.cloud.functions</groupId>
   <artifactId>functions-hello-world</artifactId>
+  <!-- [END functions_example_pom] -->
 
   <parent>
     <groupId>com.google.cloud.samples</groupId>
     <artifactId>shared-configuration</artifactId>
     <version>1.0.17</version>
   </parent>
 
+  <!-- [START functions_example_pom] -->
   <properties>
+    <!-- [END functions_example_pom] -->
     <powermock.version>2.0.7</powermock.version>
+    <!-- [START functions_example_pom] -->
     <maven.compiler.target>11</maven.compiler.target>
     <maven.compiler.source>11</maven.compiler.source>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
this is usually not required, can we remove this from the tag too?

Reply:
See #2851